nucypher/config/storages.py

```diff
@@ -345,10 +343,14 @@ def init_db_tables(self):
             self.db_conn.execute(f"CREATE TABLE {self.NODE_DB_NAME} ({node_db_schema})")
 
     def __write_node_metadata(self, node):
-        from nucypher.network.nodes import FleetStateTracker
-        node_dict = FleetStateTracker.abridged_node_details(node)
-        db_row = (node_dict['staker_address'], node_dict['rest_url'], node_dict['nickname'],
-                  node_dict['timestamp'], node_dict['last_seen'], node_dict['fleet_state_icon'])
+        node.mature()
+        node_dict = node.abridged_node_details(node)
```

Comment:
Should this be the following since it is a static method (?):

```suggestion
        node_dict = Teacher.abridged_node_details(node)
```

Reply:
Ah, I guess it may cause import issues :sigh:

Reply:
Hmm, I think Ill leave it.

Reply:
👍 - Yeah i realized after the initial comment.